### ISSUE TITLE
FR:Restore binary backup to ROS devices

### ISSUE THREAD

**Issue by hellofaduck** [eduardogsilva/routerfleet#19]
Since routerfleet has administrator rights on devices, it would be great if it was possible to restore a binary backup on devices from which it was made, and possibly on other devices as well!
It's not hard to do, just upload the file via scp and execute the command:
/system backup load name=flash/nightly_backup.backup password=""
y

**Comment by eduardogsilva** [comment]
Hello!

I intend to implement this in the near future. I'm just cautious because a wrong restore can break things or leave the router without connectivity.
I will leave this issue open for now.

Thanks

**Comment by hellofaduck** [comment]
I agree, this is a potentially dangerous action, I think some sort of safety mechanism like an additional confirmation, or an additional switch could be added to make the user confirm that they are aware of what they are doing

**Comment by j-shalom** [comment]
where does routerfleet saves the backups of the mikrotik router

**Comment by eduardogsilva** [comment]
The binary backups are stored on the docker volume: routerfleet_media_root

The default location on the docker host is:
/var/lib/docker/volumes/routerfleet_media_root/_data/backups

If you remove or change any backup on this folder, it will not reflect on the web interface and it will result in a broken link.

You can download the backups from the web interface if needed.


Cheers!

**Comment by kalazino** [comment]
Hi, 
is it possible to change the path where the backups are saved, perhaps with an NFS/SMB share? 
Thank you

**Comment by eduardogsilva** [comment]
> Hi, is it possible to change the path where the backups are saved, perhaps with an NFS/SMB share? Thank you

It's possible changing the docker storage configurations. Please, open a new discussion for this issue if you need help. 
Please, keep this topic to discuss only about backup restore function.
